grammars/puppet.cson

```diff
@@ -146,7 +146,7 @@
     'include': '#variable'
   }
   {
-    'begin': '(?i)\\b(import|include|contain)\\b\\s*'
+    'begin': '(?!.*=>)\\b(import|include|contain|require)\\s+'
```

Comment:
Looks like both work:
https://regex101.com/r/dO6zO1/1
https://regex101.com/r/nS2uE2/1

I feel like having it at the start is clearer, your call